packages/crew_url_opener.rb

```diff
@@ -0,0 +1,104 @@
+require 'package'
+
+class Crew_url_handler < Package
+  description 'Script for opening an URL in Chrome OS browser'
```

Comment:
`Script for opening an URL in your default browser`?

Reply:
Done